README.md

```diff
@@ -23,26 +26,23 @@ OuterGameSpace is a trivia game platform catered towards creating in-person game
 
 ## Usage
 
-OuterGameSpace is meant to be played with a group together in the same room. There are two different clients: Player and Presenter.
+OuterGameSpace allows trivia lovers to host games for other players to compete in.
 
-Each player connects to the game via a desktop or mobile device. Each player uses this as their own game screen. There is also a main presenter screen that needs to visible to all players that shows the game board and is used to display information about the state of the game. For example: who has answered the current question so far.
+Hosts and players connects to the game via a desktop or mobile device. Each player uses this as their own game screen. The host has a view of all players who have joined their game and their progress as they answer each question on their own time, e.g., who has answered the current question so far.
 
-Presenter client:
+This iteration of OuterGameSpace brings the presenter and host clients together into a single client that can be access at:
 http://<YOUR_OUTERGAMESPACE_SERVER>
 
-Player client:
-http://<YOUR_OUTERGAMESPACE_SERVER>/join
-
 The steps to set-up a game is as follows:
-1. Someone initiates a game by connecting to the Presenter client and selecting 'Create a New Game'
-1. The Presenter client will now display a 4-letter game code visible to everyone
-1. Players can now join the game by entering the 4-letter game code and their name into the form
-1. Accepted players will now by shown on the presenter screen
-1. The Presenter can now start the game
-1. The Player and Presenter will now display questions on the screen with a countdown timer
-1. The Player can now submit and answer to the question
-1. After all players have submitted an answer or the timer expires, the Presenter will show the correct answer and current score results
-1. After a full round of questions have occurred, the Presenter will display the final results and provide an option to restart the game
+1. All players sign in and reach the lobby of a game.
+1. Someone initiates a game by selecting 'Create Game' from the main lobby.
+1. Other players in the lobby can see when a new game is available and choose to join it if max players has not been reached for that game.
+1. Accepted players will now by shown on the presenter screen.
```

Comment:
Typo/grammatical error: should read

Accepted players will now **_be_** shown on the presenter screen.